recipes/travis-encrypt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: travis-encrypt
+  version: "0.6.0"
```

Comment:
Unless there is a compelling reason, update to the newest release (which looks like `0.7.1`).